ha/setup/ha_setup.py

```diff
@@ -323,24 +343,24 @@ def process(self):
 
         Log.info("config command is successful")
 
-    def _get_s3_instance(self, machine_id: str) -> int:
-        """
-        Return s3 instance
-
-        Raises:
-            HaConfigException: Raise exception for in-valide s3 count.
-
-        Returns:
-            [int]: Return s3 count.
-        """
-        try:
-            s3_instances = Conf.get(self._index, f"server_node.{machine_id}.s3_instances")
-            if int(s3_instances) < 1:
-                raise HaConfigException(f"Found {s3_instances} which is invalid s3 instance count.")
-            return int(s3_instances)
-        except Exception as e:
-            Log.error(f"Found {s3_instances} which is invalid s3 instance count. Error: {e}")
-            raise HaConfigException(f"Found {s3_instances} which is invalid s3 instance count.")
+    #def _get_s3_instance(self, machine_id: str) -> int:
```

Comment:
For now I kept this commented. Will remove or uncomment as per the need.

Reply:
Please remove before the merge if it is not needed.

Reply:
Sure.

Reply:
Waiting for final patch where this is removed.

Reply:
Done. Removed.